app/vep/vep_resources.py

```diff
@@ -54,12 +61,30 @@ async def submit_vep(request: Request):
         else:
             raise Exception
     except MaxBodySizeException:
-        return HTTPException(
-            status_code=status.HTTP_413_REQUEST_ENTITY_TOO_LARGE,
-            detail="Maximum file size limit exceeded.",
-        )
+        return response_error_handler(result={"status": 413})
     except Exception as e:
-        return HTTPException(
-            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
-            detail="Could not submit VEP job/workflow.",
-        )
+        return response_error_handler(result={"status": 500})
```

Comment:
Are we using integer status codes or constants (`status.HTTP_500_INTERNAL_SERVER_ERROR`)?